test/integration/features/profile.feature

```diff
@@ -2,104 +2,176 @@
+     Then stdout should match "Disk \/dev\/sda: 3[0-2]\.?[0-9]{0,2} (GB|GiB)"
+
+  Scenario: As user, I can check that cpus value was applied on profile 'foo'
+    Given Profile "foo" of Minishift has state "Running"
+     When executing "minishift ssh -- cat /proc/cpuinfo" succeeds
+     Then stdout should match "processor\s*: 0"
+      And stdout should match "processor\s*: 1"
+      And stdout should match "processor\s*: 2"
+      And stdout should match "processor\s*: 3"
+      And stdout should not match "processor\s*: [4-9]"
+
+  Scenario: As user, I can get profile 'foo' internal docker registry address
+     Given Profile "foo" of Minishift has state "Running"
+      When executing "minishift openshift registry" succeeds
+      Then stdout should be valid IP with port number
+
+  Scenario: As user, I can list profile 'foo' and 'minishift'
+    Given Profile "foo" of Minishift has state "Running"
+     When executing "minishift profile list" succeeds
+     Then stdout should match "-\s*foo\s*Running\s*\(Active\)"
+      And stdout should match "-\s*minishift\s*Running"
+   
+  Scenario: As user, I can execute command on non active running profile 'minishift' from active profile 'foo'
+    Given Profile "foo" of Minishift has state "Running"
+     When executing "minishift --profile minishift stop" succeeds
+     Then stdout should contain
+     """
+     Stopping local OpenShift cluster...
+     Cluster stopped.
+     """
+      And executing "minishift --profile minishift status" succeeds
+      Then stdout should match "Minishift:\s*Stopped\nProfile:\s*minishift\nOpenShift:\s*Stopped"
+
+  Scenario: As user, I can get status of active profile 'foo'
+    Given Profile "foo" of Minishift has state "Running"
+     When executing "minishift status" succeeds
+     Then stdout should match "Minishift:\s*Running\nProfile:\s*foo\nOpenShift:\s*Running"
+
+  Scenario: As user, I can set 'minishift' as the active profile
+    Given Profile "foo" of Minishift has state "Running"
+     When executing "minishift profile set minishift" succeeds
+     Then stdout should contain
       """
       Profile 'minishift' set as active profile
       """
 
-Scenario: User should be able to list 'minishift' as the active profile
-    When executing "minishift profile list" succeeds
-    Then stdout should contain
-    """
-    - minishift	Running		(Active)
-    """
-
-Scenario: User should be able to delete profile 'foo'
-    When executing "minishift profile delete foo --force" succeeds
-    Then stdout should contain
-    """
-    Profile 'foo' deleted successfully
-    """
-
-Scenario: User can not delete default profile 'minishift'
-    When executing "minishift profile delete minishift --force"
-    Then exitcode should equal "1"
-     And stderr should contain
-     """
-     Default profile 'minishift' can not be deleted
-     """
+  Scenario: As user, I can list 'minishift' as the active profile
```

Comment:
Scenarios should describe a general behaviour of product, here the behaviour/feature of minishift is not that it can list minishift as active profile, the feature is that minishift can list active profile, in context of previous scenarios this profile is named 'minishift' but that does not matter for whole product, this should be just checked in the steps.

Also this applies for a lot of scenario texts in this PR.